src/stories/list.stories.js

```diff
@@ -1,26 +1,22 @@
 import React from 'react';
 import List from '../Me/components/List';
 import Card from '../Me/components/Card';
+//import ListItem from '../Me/components/ListItem'
 
 import { StoriesContainer } from './StoriesContainer';
 
-export default { title: 'Profile List and List Item' };
+export default { title: 'List and List Item' };
 
-//looking for 
-// email
-// country
-// tags - programming skills
-// available
-
-export const ListA = () => {
+// List Component takes value as string or number - see example below
+export const ListProfile = () => {
     const items = [
```

Comment:
I'd suggest to move `items` on level up outside `ListProfile` so you could easly reuse it inside `ListGeneric`